### PR TITLE
Filling in the howto & trying to span the table.

### PR DESCRIPTION
**Pull request by mgifford**
Alternatively, this could just be a single line at the bottom of the table. It's ultimately the same process whatever license you pick.

### CODE REVIEW

**Comment by gcharest** [comment]
Argh, reading required on my end. I'd go with the single line at the bottom to avoid repetition.

File: en/guides/publishing-open-source-code.md [hunk 1]
```diff
@@ -85,15 +85,15 @@ Recommended permissive licences are:
 |                                          | MIT | Apache 2.0 |
 | ---------------------------------------- | --- | ---------- |
 | **When to use** | Small, simple projects and scripts. | Larger software project use Apache 2.0 because it provides a grant of patents. |
-| **How to apply to your code** |  |  |
+| **How to apply to your code** |  <td colspan=2>Add the text of your chosen lisense to a LICENSE.txt file in the root folder of your project.  |
```
Comment (nschonni) [inline en/guides/publishing-open-source-code.md]: You can't mix the `|` and HTML table syntax in GitHub. You'll need to either convert the entire table to HTML, or keep the empty cell

**Comment by gcharest** [comment]
I think simply removing the line will be sufficient?

Le lun. 29 avr. 2019 19 h 07, Nick Schonning <notifications@github.com> a
écrit :

**Comment by mgifford** [comment]
Agreed.

**Comment by smellems** [comment]
I moved the line after the two tables.

I think its supposed to be licence in Canada.  We still have a few licenses in there and I don't know if Github supports LICENCE files.

**Comment by gcharest** [comment]
Yes, Licence with a `c`

**Comment by gcharest** [comment]
Don't think it supports that spelling. Shall we try? :thinking:

**Pull request merged**